pyannote/database/custom.py

```diff
@@ -66,10 +65,16 @@ def subset_iter(database_name, file_lst=None, file_rttm=None, file_uem=None):
     # load annotations
     if file_rttm is not None:
 
-        if file_rttm.suffix == '.rttm':
+        if file_rttm.is_dir():
```

Comment:
We could make the code more readable by first building a list of paths to MDTM/RTTM files. This list would contain multiple elements when a directory is provided, and just one element when the MDTM/RTTM file is passed directly.

Then, I'd rather have the loop over all files here rather than adding complexity to `load_rttm` and `load_mdtm`. Let's keep `load_rttm` and `load_mdtm` simple (and only support one `Path` as argument).

Reply:
I implemented it this way because `pandas` has `pd.concat` function that seamlessly allow to stack and iterate over several annotation files. It means also that there is no need to change how we loop though annotations returned by pandas. 

The way you suggest will mean that we have to join different annotation dictionaries that will be returned by `load_rttm` or `load_mdtm`. Not sure it will be simpler.